packages/badge/src/FzBadge.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div :class="classes">
+    <slot></slot>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+
+const props = withDefaults(
+  defineProps<{
+    /**
+     * Color variant
+     */
+    color: "black" | "error" | "warning" | "success" | "info";
+  }>(),
+  {
+    color: "black",
+  },
+);
+
+const classes = computed(() => [
+  "text-xs px-12 rounded-xl w-fit",
+  {
```

Comment:
in piccolo ma direi che è lo stesso pattern che pero' abbiamo tolto in favore di uno switch in iconbutton no?

Reply:
sì hai ragione

Reply:
fatto in 3a1c8b5